src/FileSystemUtilities.js

```diff
@@ -51,7 +51,10 @@ export default class FileSystemUtilities {
 
   @logger.logifyAsync()
   static rimraf(filePath, callback) {
-    rimraf(filePath, callback);
+    ChildProcessUtilities.exec("npm bin", {cwd: __dirname}, (err, npmBinPath) => {
```

Comment:
Can/should we cache this?  Seems wasteful to shell out for it every time when it's constant.